config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  get 'scooters/index'
```

Comment:
don't need this anymore, redundant with the `resources` keyword, same goes for the other 3